docs/modules/hive/pages/usage-guide/security.adoc

```diff
@@ -0,0 +1,41 @@
+= Security
+
+== Authentication
+Currently, the only supported authentication mechanism is Kerberos, which is disabled by default.
+For Kerberos to work a Kerberos KDC is needed, which the users need to provide.
+The xref:home:secret-operator:secretclass.adoc#backend-kerberoskeytab[secret-operator documentation] states which kind of Kerberos servers are supported and how they can be configured.
+
+=== 1. Prepare Kerberos server
+To configure HDFS to use Kerberos you first need to collect information about your Kerberos server, e.g. hostname and port.
+Additionally, you need a service-user which the secret-operator uses to create principals for the HDFS services.
+
+=== 2. Create Kerberos SecretClass
+The next step is to enter all the necessary information into a SecretClass, as described in xref:home:secret-operator:secretclass.adoc#backend-kerberoskeytab[secret-operator documentation]. The following guide assumes you have named your SecretClass `kerberos`.
+
+=== 3. Configure HDFS to use SecretClass
+The next step is to configure your HdfsCluster to use the newly created SecretClass. Please follow the xref:hdfs:usage-guide/security.adoc[HDFS security guide] to set up and test this.
+Please make sure to use the SecretClass named `kerberos`.
```

Comment:
Can we please also document the requirement of HMS impersonating the HMS clients?

So to be concrete the following settings:
https://github.com/stackabletech/hive-operator/blob/1f4ee1b4efb6147395258ba869b85166f2908b3d/tests/templates/kuttl/kerberos-hdfs/30-install-hdfs.yaml.j2#L28-L34